### PR TITLE
Support overriding typescript version

### LINKED ISSUE #99: Support overriding typescript version
I need to be able to override the typescript version.
It would be great if compiler could be specified in options just as with ts-loader

### PR DESCRIPTION
**Pull request by Pajn**
fixes #99

### CODE REVIEW

File: src/CancellationToken.ts [hunk 1]
```diff
@@ -2,25 +2,28 @@ import crypto = require('crypto');
...
   cancellationFileName: string;
 }
 
 class CancellationToken {
+  ts: typeof tsTypes;
   isCancelled: boolean;
   cancellationFileName: string;
   lastCancellationCheckTime: number;
-  constructor(cancellationFileName: string, isCancelled: boolean) {
+  constructor(ts: typeof tsTypes, cancellationFileName: string, isCancelled: boolean) {
```
Comment (piotr-oles) [inline src/CancellationToken.ts]: I would suggest to stay with `ts` as types namespace and change name from `ts` to `compiler` for a compiler instance.

**Review by piotr-oles — CHANGES_REQUESTED** [review]

File: src/CancellationToken.ts (hunk 1 above)
Comment (Pajn) [reply src/CancellationToken.ts]: I followed the pattern in NormalizedMessage, but I have no problem changing if that's preferred.
Should I change NormalizedMessage at the same time to keep things consistent?

Comment (piotr-oles) [reply src/CancellationToken.ts]: Yes, please :)

Comment (Pajn) [reply src/CancellationToken.ts]: Done

**Review by piotr-oles — APPROVED** [review]

**Comment by johnnyreilly** [comment]
Closing due to inactivity - if this is still required please could you resubmit.

**Comment by halfnibble** [comment]
We used to specify the version of the typescript compiler with the `typescript` option. What version did this option get removed in?

**Comment by johnnyreilly** [comment]
5.0 I believe

**Pull request closed without merging**